portfolio/src/main/webapp/style.css

```diff
@@ -32,6 +32,11 @@
   font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
 }
 
+.about_me_text{
+  font-size: 16px;
+  padding-right: 23%;
```

Comment:
Make sure you're testing this with different screen sizes. A few discrete values based on media breakpoints might work better, but if it looks look across the spectrum feel free to keep it.